### ISSUE TITLE
build tasks menu selection no longer read by NVDA

### ISSUE THREAD

**Issue by Neurrone** [Microsoft/vscode#56359]
Issue Type: <b>Bug</b>

1. Press `ctrl+shift+b` to launch the build tasks menu (in my case, trying to build a rust file).
2. Pressing the arrow keys, the currently selected option is not announced. This used to work, and was probably broken only recently.

Other types of menus, such as ctrl+shift+p work fine.

VS Code version: Code - Insiders 1.27.0-insider (8727085fff33bf7d8f5e27483b10f72508d4167a, 2018-08-14T05:18:11.938Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.86GB (9.31GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|yes|
|VM|36%|

</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.19.0
vscode-eslint|dba|1.4.12
python|ms-|2018.7.1
cpptools|ms-|0.17.7
rust|rus|0.4.9
markdown-preview-enhanced|shd|0.3.5

</details>

**Comment by isidorn** [comment]
@Neurrone As far as I undestand you are reffering to the quick pick not properly announcing the list of build tasks
Forwarding to @dbaeumer and @chrmarti

**Comment by Neurrone** [comment]
@isidorn yup, that's right. Sorry I'm not sure what the correct name for the menu is.

**Comment by isidorn** [comment]
@Neurrone no problem. Yeah the large one in the middle of the screen we call Quick PIck or Quick Open.

**Comment by dbaeumer** [comment]
@chrmarti I assume that this is the quick pick I use. I never did anything special here.

**Comment by dbaeumer** [comment]
Please ping if you think otherwise.

**Comment by Neurrone** [comment]
@chrmarti as of the latest build after your commit, it works but there are some weird speech artifacts. 

The box has 2 items:

1. Rust: cargo build
2. Rust: cargo check

Some speech output:

```sh
tree view
Rust: cargo build recently used tasks   not selected  1 of 2  level 1
*press down arrow*
Rust: cargo build recently used tasks
Rust: cargo check   not selected  2 of 2  level 1
```

* it is reading the first item as not selected, even though it is highlighted.
* It does the same when pressing down arrow, but also echos the previously selected item.

I'm not sure if this is an issue with the combo box implementation, or perhaps a problem with the extension. If there's another place this widget is used, I can test it as well. Looking at the diff, I can't find any obvious causes of this behaviour, since aria-activedescendant is set. Perhaps an off by 1 error?
